api/wallet/spec/definitions/Provider.yaml

```
@@ -7,16 +8,16 @@ required:
 properties:
   id:
     x-rebillyMerge:
-      - $ref: '#/definitions/ProviderID'
+      - $ref: "#/definitions/ProviderID"
   name:
     description: |
-      Человекочитаемое наименование провайдера услуг
+      Human readable name of the service provider
```

Для унификации. Выше через дефис писали
```suggestion
      Human-readable name of the service provider
```